drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -496,18 +513,18 @@ void nrf_wifi_if_init_zep(struct net_if *iface)
 
 #ifdef CONFIG_NRF700X_STA_MODE
 	net_mgmt_init_event_callback(&ip_maddr4_cb,
-		     ip_maddr_event_handler,
-		     NET_EVENT_IPV4_MADDR_ADD | NET_EVENT_IPV4_MADDR_DEL);
+			ip_maddr_event_handler,
+			NET_EVENT_IPV4_MADDR_ADD | NET_EVENT_IPV4_MADDR_DEL);
 	net_mgmt_add_event_callback(&ip_maddr4_cb);
 
 	net_mgmt_init_event_callback(&ip_maddr6_cb,
-			     ip_maddr_event_handler,
-			     NET_EVENT_IPV6_MADDR_ADD | NET_EVENT_IPV6_MADDR_DEL);
+			ip_maddr_event_handler,
+			NET_EVENT_IPV6_MADDR_ADD | NET_EVENT_IPV6_MADDR_DEL);
 	net_mgmt_add_event_callback(&ip_maddr6_cb);
 #endif /* CONFIG_NRF700X_STA_MODE */
 #ifdef CONFIG_NRF700X_DATA_TX
 	k_work_init(&vif_ctx_zep->nrf_wifi_net_iface_work,
-		    nrf_wifi_net_iface_work_handler);
+			nrf_wifi_net_iface_work_handler);
```

Comment:
I would leave this or move to a new commit.

Reply:
Compliance check was failing when I didn't change this--would it be best to move to another commit?

Reply:
oh, so, might have crept in recently or something has changed in compliance, either way, yes please move to a new commit, thanks.

Reply:
I think I was mistaken or that these issues were only showing locally. Compliance has passed now. Thank you!